etc/templates/glidein_condor_config.template

```diff
@@ -27,7 +27,7 @@ AUTO_INCLUDE_SHARED_PORT_IN_DAEMON_LIST=False
 USE_SHARED_PORT=False
 
 # START = ( (Owner == "$USER_NAME") && (JOB_NODE_SET ==  "$NODE_SET" ) )
-START = (Owner == "$USER_NAME")
+# START = (Owner == "$USER_NAME")
```

Comment:
If not providing examples for something a user can set in a config, we probably should not have commented out code when branch is ready to merge.

Reply:
Yes the comment lingering there is kind of cryptic reminder that "we used to set this here, but now it is set elsewhere, in env vars" (in case someone would actually look for the START expression). 
So yes it can be removed,  or maybe the comment statement can actually just say that the START condition is not set in the config but in scripted Env vars.